docs/contributors/testing.md

```diff
@@ -213,6 +213,31 @@ Html reports are generated for test execution and coverage in
 - `build/test-report/e2e` for end-to-end tests
 
 
+### Web Browser choice for end-to-end tests
+
+By default, tests use Chromium and Chromium is installed by Puppeteer when running `npm install`.
+
+To use Firefox instead, first run the following command to install Firefox:
+- on Linux or macOS
```

Comment:
Maybe, we should group the all actions for a specific OS together, to avoid mistake and search who are the sections to apply 🙂

Reply:
yes, this was a first shot. I will change this by what you suggest.

Reply:
fixed in c3a8cc9